Schedule_app/Templates/bookticket.html

```diff
@@ -2,26 +2,31 @@
 
 <h1>
     Book a ticket
+    <script src="https://unpkg.com/htmx.org@1.6.0"></script>
+
 </h1>
 
+<form action = "{% url 'seats' %}" method = "POST">
+    {% csrf_token %}
 {{form.city.label_tag}}
-{% render_field form.city class="custom-select mb-4" autocomplete="off" hx-get="/cinema/" hx-target="#id_cinema"%}
+{% render_field form.city class="custom-select mb-4" autocomplete="off" hx-trigger="change, load"  hx-get="/book_ticket/cinema/" hx-target="#id_cinema"%}
```

Comment:
I am not sure about the {% url %} question and the book_ticket/book_ticket url is defined

Reply:
That's why I am asking you, because you could replace this with tags and try to see if it's working :).
Regarding the book_ticket/book_ticket, I was asking because the naming seems wrong in one of the path members if we have to repeat ourselves like this.